wire-ios/Wire-iOS/Sources/UserInterface/MainController/SidebarViewControllerDelegate.swift

```diff
@@ -112,6 +112,8 @@ final class SidebarViewControllerDelegate: WireSidebarUI.SidebarViewControllerDe
                 await mainCoordinator.showArchive()
             case .settings:
                 await mainCoordinator.showSettings()
+            case .meetings:
+                await mainCoordinator.showSettings()
```

Comment:
question: didn't you mean ?
```suggestion
                await mainCoordinator.showMeetings()
```